eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -104,213 +126,227 @@ stages:
                 dependsOn: TagRepository
 
                 pool:
-                  name: azsdk-pool-mms-ubuntu-2004-general
-                  vmImage: MMSUbuntu20.04
+                  name: $(LINUXPOOL)
```

Comment:
Offline do I recall that this failed? And it's also the first one to do so? Variable related?

Reply:
it was just create release tag one, all the other release deploy jobs worked fine for some reason

Reply:
Ah I think it fails on all deployment jobs, which I would not expect to happen. Updated others until we can debug.